### PR TITLE
fix: use JSON.stringify to cast console output rather than String

### LINKED ISSUE #34417: console.log of an array is misleading
**Describe the bug**
In the challenge pages, console.log of an array logged without brackets. It avoids to understand if array is nested or not:
```js
console.log([1,2,3,4]);
// 1,2,3,4
console.log([[1,2],[3,4]])
// 1,2,3,4
```

**To Reproduce**
1. Go to any challenge with an editor (e.g: https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/basic-data-structures/copy-an-array-with-the-spread-operator)
2. `console.log([[1,2],[3,4]])`
3. Observe the console

**Expected behavior**
Showing brackets in the console.

**Screenshots**
fcc-array-console-bug

**Desktop**
 - OS: Ubuntu 18.04 LTS
 - Browser Firefox
 - Version 63.0

_and_

 - OS: Windows 10
 - Browser Google Chrome
 - Version 70.0

### PR DESCRIPTION
**Pull request by justinsabourin**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #34417 

I have tested on a local copy.

### CODE REVIEW

**Comment by justinsabourin** [comment]
Hi @raisedadead, I ran the current master branch locally and the bug still occurs. I've pushed some changes to resolve the conflicts in the PR.

File: client/src/client/workers/test-evaluator.js [hunk 1]
```diff
@@ -4,13 +4,18 @@ import __toString from 'lodash/toString';
...
+  });
   return oldLog(...args);
 };
 
 onmessage = async e => {
   /* eslint-disable no-unused-vars */
-  const { code = '' } = e.data;
+  const {
+    code = ''
+  } = e.data;
```
Comment (ValeraS) [inline client/src/client/workers/test-evaluator.js]: @justinsabourin Is it your formatting or prettier?

**Review by ValeraS — CHANGES_REQUESTED** [review]
@justinsabourin You should add it into `buildProxyConsole` too https://github.com/freeCodeCamp/freeCodeCamp/blob/23cb7b80430d8d11b8c12214b59b883ee99fd05d/client/src/templates/Challenges/utils/frame.js#L76

**Comment by Bouncey** [comment]
@justinsabourin are you able to implement @ValeraS's suggested change?

**Review by ValeraS — APPROVED** [review]

**Comment by Bouncey** [comment]
Great work @justinsabourin 

Thank you 👍 and congratulations on your first accepted pull request to freeCodeCamp/freeCodeCamp. We hope you will find time to contribute again in the future.

:sparkles: :sparkles: :sparkles: :sparkles: :sparkles: :sparkles: :sparkles: :sparkles: :sparkles: :sparkles:

**Review by Bouncey — APPROVED** [review]

**Pull request merged**